thelounge/thelounge
# Enable in-browser run of client testing

This gets us one step closer to cross-browser (unit) testing 🎉

To test this:

- `yarn install`
- `yarn start:mocha`
- Go to http://localhost:8080/ in the browser you want to test, and you will see this:

<img width="400" alt="screen shot 2018-02-22 at 01 43 08" src="https://user-images.githubusercontent.com/113730/36523887-c576f0ce-1771-11e8-942b-2dbf76f024c7.png">

What I'm hoping then is to be able to run these steps in cloud-based browser testing solutions ([BrowserStack](https://www.browserstack.com/)? [Sauce Labs](https://saucelabs.com/)? Else?) and, as we grow our test suite, make sure that what we do is compatible with the browsers we expect compatibility with.
I haven't done this cloud-based browser testing SaaS part in the past, so this is new ground for me 🙏
